datanode/networkhistory/snapshot/orders/restore_current_orders_set.go

```diff
@@ -13,21 +13,27 @@ import (
 func RestoreCurrentOrdersSet(ctx context.Context, conn sqlstore.Connection) error {
 	// Note, testing shows that dropping the index on the current flag does not net improve performance of the following update queries
 
+	query := `WITH current_orders AS (
```

Comment:
How does this perform compared to the current query and has the state of the orders table after restore been compared to the source table state to check it's the same?